File: backend/components/metiers/threed/security/docs/samples/access/samples/mfa/sample_access.js
// sample_access.js
// Exemple d’intégration avancée pour la gestion des accès

const { generateMfaToken, provisionUser } = require('../helpers');

// Provisionnement d’un utilisateur
const user = { id: 'u123', name: 'Alice' };
const provisioned = provisionUser(user);
console.log('Provisioned:', provisioned);

// Génération d’un token MFA
const mfaToken = generateMfaToken(user.id);
console.log('MFA Token:', mfaToken);

File: backend/components/metiers/threed/security/docs/samples/access/samples/mfa/sample_access.py
# sample_access.py
# Exemple d’intégration avancée pour la gestion des accès

from helpers import generate_mfa_token, provision_user

user = {'id': 'u123', 'name': 'Alice'}
provisioned = provision_user(user)
print('Provisioned:', provisioned)

mfa_token = generate_mfa_token(user['id'])
print('MFA Token:', mfa_token)
